app/controllers/state_file/questions/md_pension_exclusion_offboarding_controller.rb

```diff
@@ -0,0 +1,17 @@
+module StateFile
+  module Questions
+    class MdPensionExclusionOffboardingController < QuestionsController
+      include OtherOptionsLinksConcern
+      def self.show?(intake)
+        if intake.filing_status_mfj? && intake.state_file1099_rs.present?
+          if !intake.primary_senior? || !intake.spouse_senior?
+            if intake.primary_proof_of_disability_submitted_no? || intake.spouse_proof_of_disability_submitted_no?
```

Comment:
@DrewProebstel and I spoke about possibly moving these methods to the model level so that they’re easier to test & the logic is easier to understand here. 

since this has to get out quickly, it doesn’t have to be addressed in this PR, but possibly another ticket.

Reply:
++